src/app/facility-descripton-mobile/facility-descripton-mobile.component.html

```diff
@@ -0,0 +1,5 @@
+<span class="row justify-content-center">
+    <h3 class="col-10 text-center">{{content?.title}}</h3>
+    <p class="col-7 text-center">{{content?.description}}</p>
+    <span class="image-wrapper"><img ngSrc="https://picsum.photos/375" width="375" height="375"  alt="{{content?.title}}"></span>
+</span>
```

Comment:
To samo co wcześniej, nie wydaje mi się że tutaj jest potrzebny ten znak zapytania jak już to bym dał wykrzyknik